docs/HACKING.md

```diff
@@ -600,3 +601,24 @@ _Note: this only limits the tests in a single file. You will also need to make s
 describe.only("NetworkTable", () => {
 it.only("displays a spinner when loading", () => {
 ```
+
+## Performance tests
+
+Performance tests use [Sitespeed.io](https://www.sitespeed.io/) and are run when
+PRs are merged.
+
+Sitespeed can also be run manually, though the tests expect a MAAS with a
+specific dataset. For best results a [local MAAS](#local-deployments) can be set
+up using [sample data](#sample-data).
+
+To run against a MAAS deployment you can use:
+
+```shell
+yarn sitespeed --browsertime.domain=[maas.ip.or.hostname]
```

Comment:
Would be great if we could just read the hostname and port values from the `MAAS_URL` env variable, we wouldn't need this extensive readme and differentiation between local and remote MAAS deployments.